ethstats/ethstats.go

```diff
@@ -76,12 +76,17 @@ type backend interface {
 // reporting to ethstats
 type fullNodeBackend interface {
 	backend
-	Miner() *miner.Miner
 	BlockByNumber(ctx context.Context, number rpc.BlockNumber) (*types.Block, error)
-	CurrentBlock() *types.Block
+	CurrentBlock() *types.Header
 	SuggestGasTipCap(ctx context.Context) (*big.Int, error)
 }
 
+// Mining
```

Comment:
If you want to add documentation, the docstring should start with the struct name, like so: 
```
// miningNodeBackend <bla bla> 
```